pinc/charsuite-math-symbols.inc

```diff
@@ -0,0 +1,48 @@
+<?php
+include_once($relPath."CharSuites.inc");
+
+use voku\helper\UTF8;
+
+$charsuite = new CharSuite("math-symbols", _("Math symbols"));
+
+$charsuite->description = _("Commonly used mathematical symbols not in Basic Latin");
+
+$charsuite->codepoints = [
+    'U+2202-U+2203',
+    'U+2207-U+2209',
+    'U+221a',
+    'U+221e',
+    'U+2220',
+    'U+2229-U+222b',
+    'U+2234-U+2235',
+    'U+2237',
+    'U+2248',
+    'U+2260-U+2261',
+    'U+2264-U+2265',
+    'U+2282-U+2283',
+    'U+2295',
+    'U+2297',
+];
+
+/*
+$charsuite->reference_urls = [
```

Comment:
I'm not sure https://www.pgdp.net/wiki/DP_Official_Documentation:Proofreading/Essential_Maths_for_Proofers is the best reference. I'd almost rather we create a similar page to https://www.pgdp.net/wiki/Symbols_collection (which could link to the Essential Maths for Proofers page) that talks about the character suite a bit more specifically.

Reply:
I can do that for this PR and just put an "Under construction" message in it for a couple of days, if that works?

Reply:
That works perfectly 👍